pom.xml

```diff
@@ -335,6 +336,20 @@
                     <artifactId>maven-deploy-plugin</artifactId>
                     <version>${maven-deploy.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-source-plugin</artifactId>
+                    <version>${maven-source-plugin.version}</version>
```

Comment:
Does this cover both open source and confluent owned private components?
if source jar is generated for private components, should this get promoted to public repos as well during the release?

Reply:
The license file check is what protects us for non open source/source available repos. This is actually why it's been difficult to get this enabled -- finding a solution that works for that and is unlikely to accidentally publish things we're not supposed to was tricky.